heron/spi/src/java/com/twitter/heron/spi/common/keys.yaml

```diff
@@ -62,6 +62,12 @@ SCHEDULER_IS_SERVICE:                heron.scheduler.is.service
 SCHEDULER_JAR:                       heron.jars.scheduler
 
 ################################################################################
+# keys for Topology Update command
+################################################################################
+TOPOLOGY_UPDATE_DEACTIVATE_SECONDS:  heron.topology.update.deactivate.seconds
+TOPOLOGY_UPDATE_REACTIVATE_SECONDS:  heron.topology.update.reactivate.seconds
```

Comment:
@billonahill - should we just keep only --config-property? IMO - it might not be a good idea to provide API since our goal long term is to remove them from being hard coded.
-